libs/components/popovers/src/lib/modules/dropdown/dropdown.component.spec.ts

```diff
@@ -1204,17 +1204,19 @@ describe('Dropdown component', function () {
       expect(item?.getAttribute('role')).toEqual('item-role-override');
     }));
 
-    it('should set the correct aria label when it is specified via SkyDefaultInputProvider', fakeAsync(() => {
-      defaultInputProvider.setValue('dropdown', 'label', 'default label');
+    it('should set the correct aria label when it is specified via SkyContentProvider', fakeAsync(() => {
+      contentDescriptorProvider.setContentDescriptor('Robert Hernandez');
```

Comment:
With your change to only overwrite context menus with the new providers you'll need to add this line to the top of both these tests.

`fixture.componentInstance.buttonType = 'context-menu';`

Reply:
🤦‍♀️